### PR TITLE
feat: Retrieve governance vc

### LINKED ISSUE #250: Retrieve governance vc

### PR DESCRIPTION
**Pull request by fqutishat**
closes #250

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: pkg/restapi/rp/operation/models.go [hunk 1]
```diff
@@ -16,8 +16,9 @@ import (
 
 // GetPresentationRequestResponse API response of getPresentationRequest.
 type GetPresentationRequestResponse struct {
-	PD  *presexch.PresentationDefinitions `json:"pd"`
-	Inv *outofband.Invitation             `json:"invitation"`
+	PD         *presexch.PresentationDefinitions `json:"pd"`
+	Inv        *outofband.Invitation             `json:"invitation"`
+	Governance json.RawMessage                   `json:"governance,omitempty"`
```
Comment (rolsonquadras) [inline pkg/restapi/rp/operation/models.go:21]: Wondering if having a suffix like `Credential` makes it clear that its a VC,

Comment (fqutishat) [reply pkg/restapi/rp/operation/models.go]: done

File: pkg/restapi/rp/operation/operations.go [hunk 2]
```diff
@@ -696,9 +697,24 @@ func (o *Operation) getPresentationsRequest(w http.ResponseWriter, r *http.Reque
 		return
 	}
 
+	var governanceVC []byte
+
+	if o.governanceProvider != nil {
+		var err error
+		governanceVC, err = o.governanceProvider.GetCredential(cr.CR.GetPayload().Client.ClientID)
```
Comment (llorllale) [inline pkg/restapi/rp/operation/operations.go:704]: Should be the RP's publicDID

Comment (fqutishat) [reply pkg/restapi/rp/operation/operations.go:704]: we retrieve by profile ID check the code

Comment (fqutishat) [reply pkg/restapi/rp/operation/operations.go:704]: @llorllale https://github.com/trustbloc/edge-adapter/blob/master/pkg/governance/provider.go#L142

Comment (llorllale) [reply pkg/restapi/rp/operation/operations.go:704]: This will need to be revised. There is a 1:1 relation between the RP's public DID and their clientID.

**Pull request merged**